### ISSUE TITLE
catalog may not be passed with URL?

### ISSUE THREAD

**Issue by volodymyrss** [oda-hub/dispatcher-app#136]
@ferrigno could you confirm that only catalog-passing case did not work, and in what way preciesly?

**Comment by volodymyrss** [comment]
I might have seen this happen long ago, so it's not new. If it is the case, we can avoid sending urls with catalogs. They may become large and difficult to pass and treat anyway, e.g. https://github.com/oda-hub/oda_api/issues/17.

**Comment by ferrigno** [comment]
The double job submission happened with OSA10.2. Indeed, I noticed in the logging that mosaic was requested and that was confusing. However, I deleted the cell when I resubmitted with OSA11.0. In this case, the job is running and apparently the catalog is passed, as there is no extra request for a Mosaic. 
The notebook contains my token, even if you have it already, I send it again through mattermost.
